datafusion/sql/src/unparser/expr.rs

```diff
@@ -1336,6 +1336,29 @@ impl Unparser<'_> {
         }
     }
 
+    /// Converts an UNNEST operation to an AST expression by wrapping it as a function call,
+    /// since there is no direct representation for UNNEST in the AST.
+    fn unnest_to_sql(&self, unnest: &Unnest) -> Result<ast::Expr> {
+        let args = self.function_args_to_sql(std::slice::from_ref(&unnest.expr))?;
+
+        Ok(ast::Expr::Function(Function {
```

Comment:
does this mean we rely on the `unnest()` function existing on whatever SQL engine happens to be running our SQL?

Reply:
Correct - this is just to generate valid sql string - `unnest(params)`.